lib/generators/rspec/controller/templates/request_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "<%= class_name.pluralize %>", <%= type_metatag(:request) %> do
```

Comment:
Copied over from `lib/generators/rspec/integration/templates/request_spec.rb`

I didn't try "very hard" to use the same template file because I am not sure if that is even wished for

Reply:
I'd recommend writing a spec for get name etc that returns success or something similarly basic.

Reply:
Yeah, figuring the details for the templates out was/is a bit difficult for me. I am sure, I would figure it out - but can you clarify the specs a bit maybe? I mean, what does the rspec-rails team think should be the convention for request specs that test a controller

Reply:
Does the request spec guide help you? https://relishapp.com/rspec/rspec-rails/docs/request-specs/request-spec

Reply:
> I mean, what does the rspec-rails team think should be the convention for request specs that test a controller

I don't use generators myself as I find that the similar lines tend to just be the naming only. There isn't that much of a semantic difference between a controller spec and a request spec, its the level of stack integration that differs. Personally I would be happy with:

```
describe "GET /<%= name.underscore.pluralize %>" do
  it "works! (now write some real specs)" do
    get "/<%= name.underscore.pluralize %>"
    expect(response).to be_success
  end
end
```

However finesse wise I think that `name.underscore.pluralize` is wrong here, because it needs to be name spaced (or not) appropriately.

Reply:
Let me know if you want any more feedback